pkg/apis/workload/v1alpha1/synctarget_types.go

```diff
@@ -83,7 +90,7 @@ type SyncTargetStatus struct {
 	Conditions conditionsv1alpha1.Conditions `json:"conditions,omitempty"`
 
 	// +optional
-	SyncedResources []string `json:"syncedResources,omitempty"`
+	SyncedResources []ResourceToSync `json:"syncedResources,omitempty"`
```

Comment:
add go doc, including who owns (= updates) this field. I.e. this is not updated by the syncer, but by the server (= controllers).